tdp/core/models/deployment_model.py

```diff
@@ -240,7 +240,6 @@ def from_operations(
                         OperationModel(
                             operation=OPERATION_SLEEP_NAME,
                             operation_order=i,
-                            host=host_name,
```

Comment:
`host` should be set to `None`  because it will change `host` value if the Operation is updated in the database.

Reply:
You're right.